lib/wpscan/web_site/sitemap.rb

```diff
@@ -0,0 +1,53 @@
+# encoding: UTF-8
+
+class WebSite
+  module Sitemap
+
+    # Checks if a sitemap.txt file exists
+    # @return [ Boolean ]
+    def has_sitemap?
+      # Make the request
+      response = Browser.get(sitemap_url)
+
+      # Make sure its HTTP 200
+      return false unless response.code == 200
+
+      # Is there a sitemap value?
+      result = response.body.scan(/^sitemap\s*:\s*(.*)$/i)
+      return true if result[0]
+      return false
+    end
+
+    # Gets a robots.txt URL
+    # @return [ String ]
+    def sitemap_url
```

Comment:
I think this should be renamed or return the parsed sitemap_url (logic from above)

Reply:
Have you got a suggestion for the function name then?
I was trying to go on other names/standards in the code.

Reply:
ok then let's just leave it. Thanks!